packages/terra-site/src/examples/icon/components/IconThemeable.jsx

```diff
@@ -0,0 +1,1134 @@
+import IconMaximize from 'terra-icon/lib/icon/IconMaximize';
+import IconMinimize from 'terra-icon/lib/icon/IconMinimize';
+import IconReply from 'terra-icon/lib/icon/IconReply';
+import IconTable from 'terra-icon/lib/icon/IconTable';
+import IconRequired from 'terra-icon/lib/icon/IconRequired';
+import IconGapChecking from 'terra-icon/lib/icon/IconGapChecking';
+import IconPersonHospital from 'terra-icon/lib/icon/IconPersonHospital';
+import IconPersonnelPerson from 'terra-icon/lib/icon/IconPersonnelPerson';
+import IconLeftPane from 'terra-icon/lib/icon/IconLeftPane';
+import IconFlowsheet from 'terra-icon/lib/icon/IconFlowsheet';
+import IconNotification from 'terra-icon/lib/icon/IconNotification';
+import IconLast from 'terra-icon/lib/icon/IconLast';
+import IconDevice from 'terra-icon/lib/icon/IconDevice';
+import IconPaperPencil from 'terra-icon/lib/icon/IconPaperPencil';
+import IconClipboard from 'terra-icon/lib/icon/IconClipboard';
+import IconZoomOut from 'terra-icon/lib/icon/IconZoomOut';
+import IconZoomIn from 'terra-icon/lib/icon/IconZoomIn';
+import IconEnvelope from 'terra-icon/lib/icon/IconEnvelope';
+import IconAllergy from 'terra-icon/lib/icon/IconAllergy';
+import IconPhoneDown from 'terra-icon/lib/icon/IconPhoneDown';
+import IconPhone from 'terra-icon/lib/icon/IconPhone';
+import IconPanelLeft from 'terra-icon/lib/icon/IconPanelLeft';
+import IconPanelRight from 'terra-icon/lib/icon/IconPanelRight';
+import IconTag from 'terra-icon/lib/icon/IconTag';
+import IconIPass from 'terra-icon/lib/icon/IconIPass';
+import IconScratchPad from 'terra-icon/lib/icon/IconScratchPad';
+import IconPharmacyReview from 'terra-icon/lib/icon/IconPharmacyReview';
+import IconBusy from 'terra-icon/lib/icon/IconBusy';
+import IconUnavailable from 'terra-icon/lib/icon/IconUnavailable';
+import IconImplant from 'terra-icon/lib/icon/IconImplant';
+import IconProtocol from 'terra-icon/lib/icon/IconProtocol';
+import IconVideoCamera from 'terra-icon/lib/icon/IconVideoCamera';
+import IconVideoCameraDisabled from 'terra-icon/lib/icon/IconVideoCameraDisabled';
+import IconNoSignal from 'terra-icon/lib/icon/IconNoSignal';
+import IconReload from 'terra-icon/lib/icon/IconReload';
+import IconRecurringEvent from 'terra-icon/lib/icon/IconRecurringEvent';
+import IconSeparate from 'terra-icon/lib/icon/IconSeparate';
+import IconMerge from 'terra-icon/lib/icon/IconMerge';
+import IconCompose from 'terra-icon/lib/icon/IconCompose';
+import IconUnread from 'terra-icon/lib/icon/IconUnread';
+import IconSwap from 'terra-icon/lib/icon/IconSwap';
+import IconMicrophone from 'terra-icon/lib/icon/IconMicrophone';
+import IconMicrophoneDisabled from 'terra-icon/lib/icon/IconMicrophoneDisabled';
+import IconOutgoingCall from 'terra-icon/lib/icon/IconOutgoingCall';
+import IconIncomingCall from 'terra-icon/lib/icon/IconIncomingCall';
+import IconMissedCall from 'terra-icon/lib/icon/IconMissedCall';
+const IconAll = () => (
+  <div>
```

Comment:
The change in this file includes adding the version, the Themeable Icons heading and fixing indentation.